src/components/node-list/node-list-row-list.js

```diff
@@ -4,6 +4,19 @@ import { connect } from 'react-redux';
 import NodeListRow, { nodeListRowHeight } from './node-list-row';
 import LazyList from '../lazy-list';
 
+// modify display of labels for modular pipelines to show nested relationship
+const getItemLabel = (item) => {
+  const layer = '・';
+  const whiteSpace = '&nbsp;&nbsp;&nbsp;&nbsp;';
+  if (item.type === 'modularPipeline') {
+    const levels = item.id.match(/\./g) ? item.id.match(/\./g).length : 0;
```

Comment:
I think it might be worth commenting on what the levels are based on here e.g. something around 'parse module depth from namespace'. Could even break it out into a specific function like `indentedLabel` for that part?

Reply:
sure I'll add on more comments about the indentation level in the comments

Reply:
@bru5 please see additional comments I have added in my latest commits